tests/cli-commands/init-tests.js

```diff
@@ -0,0 +1,107 @@
+const fs = require('fs-extra');
+const sinon = require('sinon');
+const assert = require('assert');
+
+const Command = require('../../cli-commands/commands/command');
+const InitCommand = require('../../cli-commands/commands/init/index');
+const GroupCommand = require('../../cli-commands/commands/group-command');
+const AsyncSoftExec = require('../../cli-commands/helpers/async-soft-exec');
+const definition = require('../../cli-commands/commands/init/definition');
+const fileSystemUtil = require('../../cli-commands/helpers/file-system-util');
+const directories = require('../../cli-commands/commands/init/specific/directories.json');
+const WithExampleOption = require('../../cli-commands/commands/init/options/with-example/with-example-option');
+
+describe('InitCommand', function () {
+    const TEST_DIR = "./cli-commands-test";
+    const WITH_EXAMPLE = { 'with-example': true };
+    const WITHOUT_EXAMPLE = { 'with-example': false };
+
+    let initialDir;
+    let initCommand;
+    let processSpy;
+    let processOptionsSpy;
+    let copyAllFilesFromDirToSpy;
+
+    before(async () => {
+        initialDir = process.cwd();
+        fs.mkdirSync(TEST_DIR);
+        process.chdir(TEST_DIR);
+    });
+
+    beforeEach(async () => {
+        initCommand = new InitCommand();
+        sinon.stub(AsyncSoftExec.prototype, "exec");
+        processSpy = sinon.spy(WithExampleOption, "process");
+        processOptionsSpy = sinon.spy(Command.prototype, "processOptions");
```

Comment:
Do we need this? If we have an option, "option.process" will be triggered which is enough to check